pandas/tests/arrays/test_integer.py

```diff
@@ -652,11 +652,9 @@ def test_preserve_dtypes(op):
 
     # groupby
     result = getattr(df.groupby("A"), op)()
-    expected = pd.DataFrame({
-        "B": np.array([1.0, 3.0]),
-        "C": np.array([1, 3], dtype="int64")
-    }, index=pd.Index(['a', 'b'], name='A'))
-    tm.assert_frame_equal(result, expected)
```

Comment:
fully construct the return frame as in the existing example

Reply:
Do you mean this 
a) 
```
result = df.groupby("A").op()
assert result.dtypes['C'].name == 'Int64'
``` 
b) or do you mean that I should construct the expected as:
```
expected = pd.DataFrame({	
        "B": np.array([1.0, 3.0]),
        "C": np.array([1, 3], dtype="Int64")
    }, index=pd.Index(['a', 'b'], name='A'))
tm.assert_frame_equal(result, expected)
```
Because in b) the expected constructed dataframe converts `Int64` to `int64`.

Reply:
> Because in b) the expected constructed dataframe converts Int64 to int64

The "Int64" (capital "I") is a pandas thing. You need

```
"C": integer_array([1, 3], dtype="Int64")
```

Reply:
Why was this marked resolved? Am I missing the `expected` and the `tm.assert_frame_equal`?

Reply:
No, you are correct, I missed that one...